src/test/javascript/spec/service/text-assessment.service.spec.ts

```diff
@@ -36,14 +54,14 @@ describe('TextAssessment Service', () => {
                     id: 1,
                     submitted: true,
                     type: 'MANUAL',
-                    submissionDate: '2020-07-07T14:34:25.194518+02:00',
+                    // submissionDate: '2020-07-07T14:34:25.194518+02:00',
```

Comment:
Can this be deleted?

Reply:
Yes true, it's not needed in this test suite